src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ImportService.cs

```diff
@@ -107,11 +105,11 @@ public async Task RemoveImportTableRowIfExists(Guid releaseId, string dataFileNa
                 new DatafileImport(releaseId.ToString(), dataFileName));
         }
 
-        public async Task FailImport(Guid releaseId, string dataFileName, string metaFileName,
+        public async Task FailImport(Guid releaseId, Guid? subjectId, string dataFileName, string metaFileName,
             IEnumerable<ValidationError> errors)
         {
             var importReleaseMessage =
-                JsonConvert.SerializeObject(BuildMessage(dataFileName, metaFileName, releaseId));
+                JsonConvert.SerializeObject(BuildMessage(subjectId ?? Guid.Empty, dataFileName, metaFileName, releaseId));
```

Comment:
Are there any data/meta files that exist in any environment with null Subject at the moment? Can't they just be deleted now avoiding the need for subjectId to be optional?

Reply:
Because SubjectId on File is nullable, I've changed the code to this:
```
            // Fail the import if this was a datafile upload
            await _importService.FailImport(
                releaseId,
                file.SubjectId.Value, // Data Files always have a SubjectId
                dataFileName,
                metaFileName,
                new List<ValidationError>
                {
                    new ValidationError("Files not uploaded correctly. Please delete and retry")
                }.AsEnumerable()
            );
```
Not sure if this is advisable?

Reply:
I've switched to this - which seems ok to me
```
file.SubjectId.HasValue ? file.SubjectId.Value : Guid.Empty,
```